test/e2e/tests/multiple-transactions.spec.js

```diff
@@ -95,4 +101,125 @@ describe('Confirm transactions', function () {
       },
     );
   });
+
+  it('creates multiple queued transactions, then confirms', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // initiates a transaction from the dapp
+        await openDapp(driver);
+        // creates first transaction
+        await driver.clickElement({
+          text: 'Send EIP 1559 Transaction',
+          tag: 'button',
+        });
+        await driver.waitUntilXWindowHandles(3);
+        const windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
```

Comment:
This confused me a little, can we be super explicit and call this `extensionTab` or something?